tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -200,6 +212,76 @@ static void test_provisioner_iv_update_flag_one(void)
 	PASS();
 }
 
+/** @brief Verify that the provisioner can provision a device multiple times after resets
```

Comment:
Minor
```suggestion
/** @brief Verify that the provisioner can provision a device after the device is reset.
 * Check that this can happen multiple times without requiring a reboot.
```